src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx

```diff
@@ -0,0 +1,295 @@
+---
+title: "How to use NRQL: the mechanics of querying"
+tags:
+  - Query your data
+  - 'NRQL: New Relic query language'
+  - Get started
+translate:
+  - jp
+  - kr
+metaDescription: "Learn how to query with NRQL, NRQL syntax, and how you can explore your data."
+redirects:
+---
+
+import queriesnrqlEventDefinitionsQueryBuilder from 'images/queries-nrql_screenshot-crop_event-definitions-query-builder.png'
+
+import queriesnrqlAttributeDefinitionsQueryBuilder from 'images/queries-nrql_screenshot-crop_attribute-definitions-query-builder.png'
+
+import queriesnrqlViewQueryforChart from 'images/queries-nrql_screenshot-crop_view-query-for-chart.png'
+
+import queriesnrqlNrqlExampleTimeseries from 'images/queries-nrql_screenshot-crop_nrql-example-timeseries.png'
+
+import queriesNrqlBasicExample from 'images/queries-nrql_screenshot-crop_basic-nrql-query-example.png'
+
+Before using any tool, it's best to know how to use it, and NRQL is no exception to that rule. There's a process to creating queries with NRQL, and how to structure and write NRQL queries. Understanding the rules of querying with NRQL enables you to get the most from your data. Even if you've never queried anything before, once you understand the basics you can find ways to access (almost) any data you need and visualize them in [charts](/docs/query-your-data/explore-query-data/use-charts/use-your-charts) and [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+## NRQL syntax [#syntax]
+
+Many people are familiar with SQL queries, and if you are, you'll be happy to know that NRQL has a lot of similarities. Here's a quick breakdown of the structure of a NRQL query:
```

Comment:
I love the tone, but I'd rework that first sentence a bit. Feel free to ignore this one, but I'd cut it down quite a bit:

`NRQL shares many similarities with the popular querying language SQL. Here's a quick breakdown of the structure of a NRQL query: `

Reply:
I've reworked the sentence in a slightly different way. Let me know what you think!